### DISCUSSION TITLE
Pokecube Statues

### DISCUSSION THREAD

**Discussion by Karagra** [Pokecube-Development/Pokecube-Issues-and-Wiki#751] [Ideas]
Ability to PCEdit a pokemon and change a new option called Statue from 0 to 1
This will do the following:
invulnerable:1 
canupdate: 0 
owner: 0,0,0,0
absorptionamount: infinite

While still having the pokemon have its normal head animations etc in that spot.

**Comment by Thutmose** [comment]
Starting to get to work on this: https://github.com/Pokecube-Development/Pokecube-Issues-and-Wiki/commit/c797b7ebeb9374bf339582c826282ab42c057873

Any other suggestions related to it?
